contrib/building.md

```diff
@@ -43,14 +43,15 @@ The following dependencies are required. All except Xcode can be installed by fo
 - [Xcode](https://developer.apple.com/xcode/) latest Xcode with command line tools installed
   - The latest Xcode should work, which can be downloaded from [Mac App Store](https://itunes.apple.com/us/app/xcode/id497799835?mt=12). To install older Xcode versions, [Xcodes.app](https://github.com/RobotsAndPencils/XcodesApp) is highly recommended
 - [Node.js](https://nodejs.org/en/) version 16 or later
-  - We recommend [using NVM](https://github.com/nvm-sh/nvm#installing-and-updating) to enable fast switching between Node.js & NPM versions
+  - We recommend [fnm](https://github.com/Schniz/fnm) or [NVM](https://github.com/nvm-sh/nvm#installing-and-updating) to enable fast switching between Node.js & NPM versions
```

Comment:
Sneaking in this too ... it's working great so far 🙈

Reply:
It sounds like the rest of us should try `fnm` soon ;-)